.flake8

```diff
@@ -0,0 +1,7 @@
+; Unfortunately flake8 doesn't support having configuration in pyproject.toml:
+; https://github.com/PyCQA/flake8/issues/234
+[flake8]
+ignore = E203, E266, E501, W503, F403, F401
```

Comment:
You could check out in MetaQS, how I configured flake8 there directly in the pre-commit file. This way, we have one file less :)

Reply:
done. now configured via command line args in pre-commit setup.